### DISCUSSION TITLE
Bullet Design to Reflect Expanded/Collapsed Status

### DISCUSSION THREAD

**Discussion by mdroidian** [vslinko/obsidian-outliner#63] [Ideas]
Great plugin!  Loving it.  Awesome work. 👍

My suggestion is mostly visual: 

- change the bullet design to show when list item is collapsed. (same as roam, logseq, dynalist)
- remove (or hide) the triangle in `.CodeMirror-gutter-wrapper`

I've tried to do this with CSS, but I'm not sure it's possible as the class that makes that distinction (`.CodeMirror-foldgutter-folded`, `.CodeMirror-foldgutter-open`) is nested in the `div` tree before the one I want to edit.

Example of what it could look like

![image](https://user-images.githubusercontent.com/3792666/113663807-873e5200-965f-11eb-956f-3b35413889a5.png)

**Comment by ShreyasKallingal** [comment]
That would be great. Remnote goes a step further and adds an option to change the size of the collapsed bullet based on how many children there are.

**Comment by vslinko** [comment]
@mdroidian Thank you for your idea!
Yes, looks like it cannot be done with CSS only.
May be it could be done with JS, but It should be researched first.

**Reply by mdroidian** [reply]
I'm not very familiar with JS, but if your plugin already knows when the list is expanded / collapsed, then adding a class of `.folded` / `.open` anywhere on this tree should suffice for CSS to do the rest of the work.

```
<pre class="HyperMD-list-line HyperMD-list-line-1 CodeMirror-line" role="presentation">
  <span role="presentation">
    <span class="cm-formatting cm-formatting-list cm-formatting-list-ul cm-list-1">
```

**Reply by mdroidian** [reply]
I also added a post to Obsidian Forum because I think these classes should be a feature in core

https://forum.obsidian.md/t/add-folded-open-to-list-node-in-edit-mode
